packages/helper-grammar-regex-collection/index.js

```diff
@@ -11,7 +11,7 @@ const importMembers = regex`[\r\n\s\w{},*\$]*`;
 const from = regex`\s from \s`;
 
 const REQUIRE = regex`
-  ( require(\.resolve)? | import )
+  ( require(\.resolve)? | import(\.resolve)? )
```

Comment:
My fault, I was fixing the implementation according to the provided tests without double checking. I can fix it tomorrow